lib/web_ui/lib/ui_web/src/ui_web/flutter_views_proxy.dart

```diff
@@ -17,13 +17,15 @@ class FlutterViewManagerProxy {
   // The proxied viewManager instance.
   final FlutterViewManager _viewManager;
 
-  /// Returns the `hostElement` configuration value passed from JS when `viewId` was added.
+  /// Returns the host element for [viewId].
+  /// In a multi-view app, it is the `hostElement` value passed from JS when [viewId] was added.
+  /// In an implicit view app, it is either the provided `hostElement` or the <body> element.
```

Comment:
The `implicitView`-based element embedding is superseded by multi-view, so I wouldn't bother documenting it. Instead, we will deprecate it and remove it so we don't have multiple ways to do the same thing.

As for multi-view, we don't want to call it "multi-view app". Instead, we'll have full-page mode and add-to-app mode, and being able to add multiple view in the add-to-app mode is simply one of many capabilities of that mode. I'd reword this paragraph to reflect that, such as:

```
/// In the full-page mode, the host element is the `<body>` element of the page
/// and the view is the one and only [PlatformDispatcher.implicitView].
///
/// In the add-to-app mode, the host element is the value of `hostElement`
/// provided when creating the view.
```

Reply:
This is much better, updated!